app/views/groups/show.html.erb

```diff
@@ -2,6 +2,7 @@
   <p style="color: green"><%= notice %></p>
 
 <div class="login-top">
+    <%= link_to "<-", groups_path, class: "login-links" %>
```

Comment:
## [OPTIONAL]
- [ ] Nice job on implementing the change however I would suggest using icons like(fontawsome, ...) so that your design can look better. check this [link](https://fontawesome.com/docs/web/use-with/rails-turbolinks)